<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_citra_awal">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CITRA AWAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_citra_hasil">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CITRA HASIL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIdentifikasi">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifikasi</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionMean_Filter"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperasi_Titik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionMean_Filtering">
   <property name="text">
    <string>Mean Filtering</string>
   </property>
  </action>
  <action name="actionEqualization_Histogram">
   <property name="text">
    <string>Equalization Histogram</string>
   </property>
  </action>
  <action name="actionMean_Filter">
   <property name="text">
    <string>Mean Filtering</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
